PWGCF/Flow/Tasks/FlowGFWPbPb.cxx

```diff
@@ -341,6 +359,33 @@ struct FlowGFWPbPb {
       // use this cut at low multiplicities with caution
       return 0;
     }
+    float vtxz = -999;
+    if (collision.numContrib() > 1) {
+      vtxz = collision.posZ();
+      float zRes = TMath::Sqrt(collision.covZZ());
+      if (zRes > 0.25 && collision.numContrib() < 20)
+        vtxz = -999;
+    }
+
+    auto multNTracksPV = collision.multNTracksPV();
+
+    if (centrality >= 70. || centrality < 0)
+      return 0;
```

Comment:
Why not returning `False` instead, it will be more readable

Reply:
You mean that to obtain collisions from 0 to 70 % centrality I should return 1 ?